src/clojure_tic_tac_toe/computer_move.clj

```diff
@@ -0,0 +1,10 @@
+(ns clojure-tic-tac-toe.computer_move
+  (:require [clojure-tic-tac-toe.board :as board]
+            [clojure-tic-tac-toe.utilities :refer [set-to-sorted-list-or-nil]]))
+
+(defn make-random-move
+  [board]
+  (let [available-moves (board/get-available-moves board)
+        sorted-list (set-to-sorted-list-or-nil available-moves)]
+    (rand-nth sorted-list)))
```

Comment:
How come you had a sort the list if you're grabbing a random element from the list?

Reply:
Technically I didn't have to, but I just decided to sort it now, because I thought it might be helpful in the future.

Reply:
That's a bit confusing to me. One of the benefits of doing Test Driven Development is that you code pretty much the simplest thing you need. I'm not sure what led you to write a test for sorting in this case. Let's delete it for now

Reply:
Fixed